lib/consumers/bvc_pergeovisibility.c

```diff
@@ -1184,15 +1496,13 @@ static int compute_geo_pfx_visibility(bvc_t *consumer, bgpview_iter_t *it)
   bgpstream_peer_sig_t *sg;
 
   /* for each prefix in the view */
-  for (bgpview_iter_first_pfx(it, BGPSTREAM_ADDR_VERSION_IPV4,
-                              BGPVIEW_FIELD_ACTIVE); //
+  for (bgpview_iter_first_pfx(it, 0, BGPVIEW_FIELD_ACTIVE); //
        bgpview_iter_has_more_pfx(it);                //
        bgpview_iter_next_pfx(it)) {
 
     pfx = bgpview_iter_pfx_get_pfx(it);
 
     /* only consider (ipv4) prefixes mask is longer than a /6 */
```

Comment:
not a blocker, but you guys might want to think about whether there are v6 prefixes longer than a /6 but are as equally silly as an ipv4 /6 prefix.

Reply:
Yeah, we had some initial discussions about this when I saw a v6 /20 during testing but need to take the time to look into some of these bigger prefixes and work out what is legit and what is bogus.